Modules/_pickle.c

```diff
@@ -4693,7 +4693,7 @@ This takes a binary file for writing a pickle data stream.
 
 The optional *protocol* argument tells the pickler to use the given
 protocol; supported protocols are 0, 1, 2, 3, 4 and 5.  The default
-protocol is 4. It was introduced in Python 3.4, and is incompatible
+protocol is 5. It was introduced in Python 3.4, and is incompatible
```

Comment:
Same for all mentions of being introduced in 3.4:
```suggestion
protocol is 5. It was introduced in Python 3.8, and is incompatible
```

Reply:
I suggest changing in [line 159](https://github.com/python/cpython/blob/main/Doc/library/pickle.rst?plain=1#L159)

> It is the default protocol starting with Python 3.8.

to

> This was the default protocol in Python 3.8--3.14.

Likewise, adding to [line 161](https://github.com/python/cpython/blob/main/Doc/library/pickle.rst?plain=1#L161)
> It is the default protocol starting with Python 3.14.

Reply:
> I suggest changing in [line 159](https://github.com/python/cpython/blob/main/Doc/library/pickle.rst?plain=1#L159)
> 
> > It is the default protocol starting with Python 3.8.
> 
> to
> 
> > This was the default protocol in Python 3.8--3.14.
> 
> Likewise, adding to [line 161](https://github.com/python/cpython/blob/main/Doc/library/pickle.rst?plain=1#L161)
> 
> > It is the default protocol starting with Python 3.14.

makes senses, I've added the suggested changes!